metadata/hammer.json

```diff
@@ -1,5 +1,7 @@
 {
   "file": "https://curriculum.code.org/media/uploads/touch.mp3",
+  "artist": "MC Hammer",
```

Comment:
We have deprecated the use of metadata files in dance-party (other than our test songs - synthesize and jazzy_beats) so we don't need to update these files and they should, at some point, be removed from the repo.

Reply:
Should I pull the extras out as part of the PR?

Reply:
We'll want to at some point, but that can come in a follow-up PR at some point.